pkg/controller/routes/sync.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2019 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routes
+
+import (
+	"context"
+
+	routev1 "github.com/openshift/api/route/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	logf "github.com/jetstack/cert-manager/pkg/logs"
+)
+
+func (c *routeRequestManager) Sync(ctx context.Context, instance *routev1.Route) (err error) {
+	log := logf.FromContext(ctx)
+
+	log.Info("Reconciling Route", "route", instance)
+
+	if instance.Spec.TLS == nil {
+		return nil
+	}
+	secretName, ok := instance.GetAnnotations()[certAnnotation]
+	caSecretName, okCa := instance.GetAnnotations()[destCAAnnotation]
+	shouldUpdate := false
+
+	if !ok {
```

Comment:
I find `ok` to be used a few lines ahead quite confusing: either it should change the name or move the variable declaration inside the if statement (if possible)